sbc_compassion/models/correspondence.py

```diff
@@ -965,4 +966,18 @@ def json_to_data(self, json, mapping_name=None):
                 pages.append(orm_tuple)
             odoo_data["page_ids"] = pages or False
 
+        # Concatenate the local_id and the current resubmit_id
+        if "local_id" in odoo_data:
+            odoo_data["local_id"] = odoo_data["local_id"] + self.resubmit_id
+
```

Comment:
No, this is in the wrong method, you should do this in the method `data_to_json` to change the value we are sending to GMC.